cmd/commands/subvolume.go

```diff
@@ -40,11 +40,10 @@ var listCmd = &cobra.Command{
 }
 
 var deleteCmd = &cobra.Command{
-	Use:                "delete",
-	Short:              "Deletes a stale subvolume.",
-	DisableFlagParsing: true,
-	Args:               cobra.ExactArgs(3),
-	Example:            "kubectl rook-ceph delete <subvolumes> <filesystem> <subvolumegroup>",
+	Use:     "delete",
+	Short:   "Deletes a stale subvolume.",
+	Args:    cobra.ExactArgs(3),
+	Example: "kubectl rook-ceph delete <filesystem> <subvolume> <subvolumegroup>",
```

Comment:
Can the `subvolumegroup` be optional and we default to `csi`?

Reply:
Mostly the subvolumegroup is named csi only, but IMO, let the user pass the subvolumegroup.  What's your opinion on this @Madhu-1 ?

Reply:
default to `csi` looks good and make this optional